SQRLDotNetClientUI/Views/MessageBox.xaml.cs

```diff
@@ -0,0 +1,186 @@
+            this.Message = Message;
+            switch (messageBoxSize)
+            {
+                case MessageBoxSize.XLarge:
+                    this.Height = 280;
+                    this.Width = 800;
+                    this.MaxHeight = 60;
+                    break;
+                case MessageBoxSize.Large:
+                    this.Height = 180;
+                    this.Width = 600;
+                    this.MaxHeight = 60;
+                    break;
+                case MessageBoxSize.Small:
+                    this.Height = 180;
+                    this.Width = 200;
+                    this.MaxHeight = 60;
+                    break;
+                default:
+                    {
+                        this.Height = 180;
+                        this.Width = 400;
+                        this.MaxHeight = 60;
+                    }
+                    break;
+            }
+
+            internalIcon = messageBoxIcon switch
+            {
+                MessageBoxIcons.ERROR => "resm:SQRLDotNetClientUI.Assets.Icons.error.png",
+                MessageBoxIcons.WARNING => "resm:SQRLDotNetClientUI.Assets.Icons.warning.png",
+                _ => "resm:SQRLDotNetClientUI.Assets.Icons.ok.png",
+            };
+            switch (messageBoxButtons)
+            {
+                case MessageBoxButtons.OKCancel:
+                {
+                        AddButton("OK", _loc.GetLocalizationValue("BtnOK"), MessagBoxDialogResult.OK);
+                        AddButton("Cancel", _loc.GetLocalizationValue("BtnCancel"), MessagBoxDialogResult.CANCEL);
+
+                }
+                break;
+                case MessageBoxButtons.YesNo:
+                    {
+                        AddButton("Yes", _loc.GetLocalizationValue("BtnYes"), MessagBoxDialogResult.YES);
+                        AddButton("No", _loc.GetLocalizationValue("BtnNo"), MessagBoxDialogResult.NO);
+                    }
+                    break;
+                default:
+                    {
+                        AddButton("OK", _loc.GetLocalizationValue("BtnOK"), MessagBoxDialogResult.OK);
+                    }
+                    break;
+
+            }
+
+            Init();
+        }
+
+        private void AddButton(string name, string content, MessagBoxDialogResult clickResult)
+        {
+            var msgWindow = AvaloniaLocator.Current.GetService<MessageBox>();
+            var panel = msgWindow.FindControl<StackPanel>("pnlButtons").Children;
+            Button btn = new Button
+            {
+                Content = content,
+                Name = name,
+                Margin = new Thickness(10, 0),
+                Width = 60
+            };
+            btn.Click += (s,e) => msgWindow.Close(clickResult); 
+            panel.Add(btn);
+        }
+
+        private void Btn_Click(object sender, Avalonia.Interactivity.RoutedEventArgs e)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
This isn't used I guess?

Reply:
Yup, I used ti to get the signature for the event and forgot to remove it.. Fixed Thanks!